lighthouse-core/lib/i18n/locales/en-US.json

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
is this file significantly shorter than the existing one?

Reply:
It is indeed!  It has no descriptions.  Only messages like all other locales.  Since it is processed like them and not uploaded to translators it does not need descriptions (only pre-locales/en-US.json does).